code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -84,21 +84,6 @@
 	. = ..()
 	disconnect_from_area(area_to_unregister)
 
-///adds a gas or list of gases to our filter_types. used so that the scrubber can check if its supposed to be processing after each change
-/obj/machinery/atmospherics/components/unary/vent_scrubber/proc/add_filters(filter_or_filters)
```

Comment:
What does this have to do with the PR?

Reply:
I added a proc called ``add_filters`` and this was using the same name and was completely unused so i just deleted it